### PR TITLE
George - CHARMS - Automation Scenario 1

### PR DESCRIPTION
**Pull request by georgeb79**

### CODE REVIEW

**Review by Mariachaudhry — APPROVED** [review]

**Review by alenaminsk — APPROVED** [review]

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/myRASScreenerSubmissions.feature [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+		And logs in via Okta with username "charmsras5@yahoo.com" and password "RASTest2023$$"
+		And clicks on Eligibility Questionnaire to begin questionnaire
+		When the participant submits a screener from excel sheet "screenerScenario1"
+		Then data submitted for scenario is verified in native view against corresponding scenario from the excel sheet
+		Given a participant is on the RASopathies Longitudinal Cohort Study login page "myRASLoginPage"
+		And logs in via Okta with username "charmsras5@yahoo.com" and password "RASTest2023$$"
+		And the participant submits a Individual Information Questionnaire for excel sheet "IIQScenario1"
 
-	@bucurgb @myRasScreenerSubmissionOnly @Updating
+	@juarezds @myRasScreenerSubmissionOnly @InProgress @Progression
```
Comment (zamant2) [inline src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/myRASScreenerSubmissions.feature:20]: InProgress and Progression tag shouldn't merge to master

File: src/test/java/CustomBusiness/CCR/Steps/InternalUserSteps.java [hunk 2]
```diff
@@ -171,7 +171,7 @@ public void user_enters_into_a_business_zip_field(String businessZip) {
     }
 
     @Given("User uploads {string} document")
-    public void user_uploads_document(String document) throws TestingException, AWTException {
+    public void user_uploads_document(String document){
```
[the remark this answers is not in the document]
Comment (georgeb79) [reply src/test/java/CustomBusiness/CCR/Steps/InternalUserSteps.java]: I added  'throws TestingException' back

File: src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/myRASScreenerSubmissions.feature (hunk 1 above)
Comment (georgeb79) [reply src/test/java/ServiceNow/CHARMS/Features/RasopathyStudy/myRASScreenerSubmissions.feature]: @InProgress was left there by Diego. This is a huge scenario and we will be adding to it a lot more. I removed the @Progression

**Review by zamant2 — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
LGTM

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**